iceberg/iceberg-handler/src/test/results/positive/show_iceberg_materialized_views.q.out

```diff
@@ -229,7 +229,7 @@ shtb_aggr_view2     	Yes                 	Manual refresh      	Available in pres
 shtb_full_view2     	Yes                 	Manual refresh (Valid for 5min)	Available in presence of insert operations only
 shtb_test1_view1    	No                  	Manual refresh      	Unknown             
 shtb_test1_view2    	Yes                 	Manual refresh (Valid always)	Available in presence of insert operations only
-shtb_view_on_native 	Yes                 	Manual refresh      	Available           
+shtb_view_on_native 	Yes                 	Manual refresh      	Available in presence of insert operations only
```

Comment:
nit: This is an existing message, but there's a missing 'the':  'Available in the presence of ..'   (a quick grammar check can be done on grammarly.com)
Or if you want to shorten it: 'Available for insert operations only'

Reply:
Changed to the short one.